x/ibc/testing/coordinator.go

```diff
@@ -244,6 +245,39 @@ func (coord *Coordinator) AcknowledgementExecuted(
 	)
 }
 
+// RelayPacket receives a channel packet on counterparty, queries the ack
+// and acknowledges the packet on source. The clients are updated as needed.
+func (coord *Coordinator) RelayPacket(
+	source, counterparty *TestChain,
+	sourceClient, counterpartyClient string,
+	packet channeltypes.Packet, ack []byte,
+) error {
+	// get proof of packet commitment on source
+	packetKey := host.KeyPacketCommitment(packet.GetSourcePort(), packet.GetSourceChannel(), packet.GetSequence())
+	proof, proofHeight := source.QueryProof(packetKey)
+
+	recvMsg := channeltypes.NewMsgRecvPacket(packet, proof, proofHeight, counterparty.SenderAccount.GetAddress())
+
+	// receive on counterparty and update source client
+	if err := coord.SendMsgs(counterparty, source, sourceClient, recvMsg); err != nil {
+		return err
+	}
+
+	// get proof of acknowledgement on counterparty
+	packetKey = host.KeyPacketAcknowledgement(packet.GetDestPort(), packet.GetDestChannel(), packet.GetSequence())
+	proof, proofHeight = counterparty.QueryProof(packetKey)
+
+	// TODO: add a query for the acknowledgement by events
+	// - https://github.com/cosmos/cosmos-sdk/issues/6509
+
+	ackMsg := channeltypes.NewMsgAcknowledgement(packet, ack, proof, proofHeight, source.SenderAccount.GetAddress())
+	if err := coord.SendMsgs(source, counterparty, counterpartyClient, ackMsg); err != nil {
+		return err
+	}
```

Comment:
Can Recv and Ack be split into two exported helper functions so they can be used independently by tests?